components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -367,7 +367,8 @@ func certValidator(cm *corev1.ConfigMap, dataKey string, log logr.Logger) error
 	odhCertData, ok := cm.Data[dataKey]
 	if !ok || odhCertData == "" {
 		// Print a warning if odh-ca-bundle.crt data is empty
-		return fmt.Errorf("Warning: %s data is empty", dataKey)
+		log.Info(fmt.Sprintf("Warning: %s data is empty", dataKey))
+		return nil
```

Comment:
This would cause problem, as line 301 and  316 
would be triggered, include something which is not present.